README.md

```diff
@@ -73,3 +73,7 @@ Builds the app for production to the `build` folder:
 ```sh
 $ yarn build
 ```
+
+## Production
+
+https://witty-ocean-02e42dd0f.azurestaticapps.net/
```

Comment:
@picuron do we not have a proper domain name yet?

Reply:
We are going to route to the Vaccine Hunters website through their DNS, so no need to get our own domain.